### PR TITLE
NCL-1590 - Add BuildConfigurationId to the BuildExecutionConfiguratio…

### PR DESCRIPTION
**Pull request by vibe13**
…n object that stores info for Running Builds

### CODE REVIEW

**Comment by vibe13** [comment]
@alexcreasy @matejonnet Hi guys, please review. Thanks!

File: build-executor/src/main/java/org/jboss/pnc/executor/DefaultBuildExecutionConfiguration.java [hunk 1]
```diff
@@ -36,17 +36,11 @@
...
-            Integer userId,
-            String buildScript,
-            String name,
-            String scmMirrorRepoURL,
-            String scmRepoURL,
-            String scmMirrorRevision,
-            String scmRevision, BuildType buildType) {
+    public DefaultBuildExecutionConfiguration(int id, String buildContentId, Integer userId, String buildScript, String name,
+            String scmMirrorRepoURL, String scmRepoURL, String scmMirrorRevision, String scmRevision, BuildType buildType,
+            int buildConfigDBId) {
```
Comment (matejonnet) [inline build-executor/src/main/java/org/jboss/pnc/executor/DefaultBuildExecutionConfiguration.java]: Could you move parameters to new lines to keep it readable.

File: build-executor/src/main/java/org/jboss/pnc/executor/DefaultBuildExecutionConfiguration.java [hunk 2]
```diff
@@ -58,6 +52,7 @@ public DefaultBuildExecutionConfiguration(
         this.scmMirrorRevision = scmMirrorRevision;
         this.scmRevision = scmRevision;
         this.buildType = buildType;
+        this.buildConfigDBId = buildConfigDBId;
```
Comment (matejonnet) [inline build-executor/src/main/java/org/jboss/pnc/executor/DefaultBuildExecutionConfiguration.java]: I would leave DB out of name, as from the executor perspective it doesn't have to do anything with db.

**Comment by vibe13** [comment]
@matejonnet made changes as suggested

**Comment by vibe13** [comment]
@matejonnet after chat on IRC, this is less intrusive!

**Pull request merged**